lib/Translator.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
```

Comment:
because you dropped all the rest of the documentation, so one way or another, the doc is needed.

Feel free to contact me on discord: Tobiah#8452, it might be easier on instant messaging.